packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -52,7 +52,11 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Initializes the camera on the device.
-  Future<void> initializeCamera(int cameraId) {
+  ///
+  /// [imageFormatGroup] is used to specify the image formatting used.
+  /// On Android this defaults to ImageFormat.YUV_420_888 and applies only to the imageStream.
+  /// On iOS this defaults to kCVPixelFormatType_32BGRA.
+  Future<void> initializeCamera(int cameraId, {int imageFormatGroup}) {
```

Comment:
Overlooked this earlier, but I think the `imageFormatGroup` should be an enum value limiting the possible values that are allowed (this will then also match with the API documentation).